.github/workflows/release.yml

```diff
@@ -387,7 +388,11 @@ jobs:
         with:
           dotnet-version: ${{ matrix.dotnet-version }}
       - run: mkdir -p ${{ runner.temp }}/opt/pulumi/nuget
-      - run: dotnet nuget add source ${{ runner.temp }}/opt/pulumi/nuget
+      - name: Add NuGet packages as a local NuGet source
+        run: |
+          echo $(which dotnet)
```

Comment:
Is `echo $(which dotnet)` the same as `which dotnet`?

Reply:
Or this is to avoid non-0 exit code terminating the workflow.

Reply:
As a habit yeah, I put diagnostics in echo statements.